packages/eslint-plugin/tools/generate-configs.ts

```diff
@@ -2,8 +2,11 @@ import { TSESLint } from '@typescript-eslint/experimental-utils';
 import chalk from 'chalk';
 import fs from 'fs';
 import path from 'path';
+import { format, Options } from 'prettier';
 import rules from '../src/rules';
 
+const prettierConfig: Options = require('../../../.prettierrc.json');
```

Comment:
Prettier exports a resolveConfig utility which can be used instead of this manual resolution (and therefore the file wouldn't need to be renamed either)